open_academy/wizards/wizard_session.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class WizardSession(models.TransientModel):
+    _name = 'wizard_session'
```

Comment:
for name of model please use "." instead of "_" also the name of the Wizard in the class and name file does not give me a hint about what this wizard if for. 
You could use a name like " add.attendee.sessions¨